spec/index.html

```diff
@@ -8,23 +8,22 @@
     1. Return *undefined*.
   </emu-alg>
   <emu-note>
-    <p>This method is designed for programs implementing spinlocks, such as those inside of mutexes, to provide a hint to the CPU that it is spinning while waiting on a value. It has no observable behaviour other than timing.</p>
+    <p>This method is designed for programs implementing spin-wait loops, such as spinlock fast paths inside of mutexes, to provide a hint to the CPU that it is spinning while waiting on a value. It has no observable behaviour other than timing.</p>
     <p>Implementations are expected to implement a pause or yield instruction if the best practices of the underlying architecture recommends such instructions in spin loops. For example, the <a href="https://www.intel.com/content/www/us/en/content-details/671488/intel-64-and-ia-32-architectures-optimization-reference-manual-volume-1.html">Intel Optimization Manual</a> recommends the <code>pause</code> instruction.</p>
   </emu-note>
   <emu-note>
-    <p>Due to the overhead of function calls, it is reasonable that an inlined call to this method in an optimizing compiler waits a different amount of time than an uninlined call.</p>
+    <p>The _N_ parameter controls how long an implementation pauses. Larger values results in longer waits. Implementations are encouraged to have an internal upper bound on the maximum amount of time paused on the order of tens to hundreds of nanoseconds.</p>
+    <p>The user, if passing the _N_ parameter, can use it to implement backoff strategies. For backoff strategies where subsequent waits become longer, linear backoff can be implemented by passing linearly increasing values for non-negative _N_; exponential backoff can be implemented by exponentially increasing values for non-negative _N_, and so on. For backoff strategies where subsequent waits become shorter, linear backoff can be implemented by passing linearly decreasing values for negative _N_; exponential backoff where subsequent waits become longer can be implemented by exponentially decreasing values for negative _N_, and so on.</p>
   </emu-note>
   <emu-note>
-    <p>It is common practice for spin loops to have backoff. For this purpose, _iterationNumber_ is a hint parameter. For integral numbers _N_, <code>Atomics.pause(N)</code> should wait at most as long as <code>Atomics.pause(N+1)</code>. The programmer is expected to linearly increase _iterationNumber_, but an implementation may choose to back off superlinearly.</p>
+    <p>Due to the overhead of function calls, it is reasonable that an inlined call to this method in an optimizing compiler waits a different amount of time than an uninlined call.</p>
```

Comment:
```suggestion
    <p>Due to the overhead of function calls, it is reasonable that an inlined call to this method in an optimizing compiler waits a different amount of time than a non-inlined call.</p>
```

"uninlined" sounds like it refers to a process of "uninlining", but we're just trying to say the absence of inlining.